integration-test/merge_test.js

```diff
@@ -12,9 +13,9 @@ const TEST_NAMESPACE = 'scratch.merge-test'
 const INVALID_STATEMENT_NAMESPACE = 'scratch.merge-test.invalid-stmt'
 
 const seedObjects = [
-  {hello: 'world'},
-  {foo: 'bar'},
-  {etc: 'and so on'}
+  {id: uuid.v4(), hello: 'world'},
```

Comment:
Curious, was there a specific reason for doing this? Are the tests noticeably faster this way?